.github/workflows/release.yml

```
@@ -0,0 +1,53 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  release:
+    env:
+      CHROME_EXTENSION_ID: dclfaikcemljbaoagjnedmlppnbiljen
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      - name: Setup Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: 16
+      - name: Install Dependencies
+        shell: bash
+        run: npm install
+      - name: Set environment variables from the manifest
+        run: |
+          echo "FIREFOX_EXTENSION_ID=$(jq -s -r ".[0].browser_specific_settings.gecko.id" < src/manifest_firefox.json)" >> "$GITHUB_ENV"
+          echo "CURRENT_VERSION=$(jq -s -r '.version' < src/manifest.json)" >> "$GITHUB_ENV"
```

Tried this locally - seems to be missing the `[0]`:

```suggestion
          echo "CURRENT_VERSION=$(jq -s -r '.[0].version' < src/manifest.json)" >> "$GITHUB_ENV"
```